src/JointAxis.cc

```diff
@@ -142,11 +143,12 @@ Errors JointAxis::Load(ElementPtr _sdf)
   {
     sdf::ElementPtr limitElement = _sdf->GetElement("limit");
 
-    this->dataPtr->lower = limitElement->Get<double>("lower", -1e16).first;
-    this->dataPtr->upper = limitElement->Get<double>("upper", 1e16).first;
-    this->dataPtr->effort = limitElement->Get<double>("effort", -1).first;
+    const double kInf = std::numeric_limits<double>::infinity();
+    this->dataPtr->lower = limitElement->Get<double>("lower", -kInf).first;
```

Comment:
You could pass the variable itself here as the default value, so if it ever changes again, it only needs to be changed on line 64, i.e.

```suggestion
    this->dataPtr->lower = limitElement->Get<double>("lower", this->dataPtr->lower).first;
```

Reply:
I've deduplicated the default values used in `JointAxis::Load` in https://github.com/gazebosim/sdformat/pull/1112/commits/c7feca99e73fdc684a9bc4705510274294d09ef5